bootstraps/stage1/spack.in.yaml

```diff
@@ -0,0 +1,28 @@
+# This is a Spack Environment file.
+#
+# It describes a set of packages to be installed, along with
+# configuration settings.
+spack:
+  specs:
+  - binutils +stage0
```

Comment:
`stage1/spack.yaml` with `+stage0` is confusing ;p

Reply:
I agree, _sigh_ iterative naming is not good naming.